src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Markdown.cs

```diff
@@ -31,13 +31,40 @@ public static DiagnosticData AddMarkdownView(this Response response, string mark
             return response.AddMarkdownView(markdown, false, title);
         }
 
+        /// <summary>
+        /// Adds markdown section to the response
+        /// </summary>
+        /// <param name="response">Response object</param>
+        /// <param name="isContainerNeeded">If true, will keep the container of markdown view</param>
+        /// <param name="markdown">String that will be translated to markdown in UI</param>
+        /// <param name="title">Title of markdown section</param>
+        /// <example> 
+        /// This sample shows how to use <see cref="AddMarkdownView"/> method.
+        /// <code>
+        /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext cxt, Response res)
+        /// {
+        ///     var markdown = @"
+        ///     ## This is header
+        ///     ";
+        ///     
+        ///     res.AddMarkdownView(markdown, "Title of markdown section");
+        /// }
+        /// </code>
+        /// </example>
+
+        public static DiagnosticData AddMarkdownView(this Response response, bool isContainerNeeded, string markdown, string title = null)
```

Comment:
nitpik: Please make the parameter order consistent with the other overloaded method. `AddMarkdownView(this Response response, string markdown, bool isContainerNeeded, string title = null)`

Reply:
Maybe you can make `isContainerNeeded` an optional parameter: `isContainerNeeded = true`. That way people can just focus on the markdown content but if they want to remove the container they can specify `false`

Reply:
This will throw an ambiguous build error : "error CS0121: The call is ambiguous between the following methods or properties: 'ResponseMarkdownExtension.AddMarkdownView(Response, string, string)' and 'ResponseMarkdownExtension.AddMarkdownView(Response, string, string, bool)'" if I make that optional as well..

Reply:
Oh okay then I'd recommend removing the parameter and if people want to remove the container they can use the other overloaded method. It would also be a good idea because changing this signature would break detectors.

Reply:
Oh okay then I'd recommend removing the parameter and if people want to remove the container they can use the other overloaded method. It would also be a good idea because changing this signature would break detectors.

Reply:
Updated PR...

Reply:
Did we update this @cindylovescoding ?